### ISSUE TITLE
NoClassDefFound ProxyClass

### ISSUE THREAD

**Issue by austin4195** [plan-player-analytics/Plan#2237]
### Describe the issue

When starting up Mohist 1.12.2-292, an error is presented after downloading the MySQL driver.

### Exceptions & other logs

```text
https://www.toptal.com/developers/hastebin/opurequnot.md
```

```
[04:07:24] [Server thread/INFO] [Mohist]: [Plan] 
[04:07:24] [Server thread/INFO] [Mohist]: [Plan]            ██▌
[04:07:24] [Server thread/INFO] [Mohist]: [Plan]      ██▌   ██▌
[04:07:24] [Server thread/INFO] [Mohist]: [Plan]   ██▌██▌██▌██▌  Player Analytics
[04:07:24] [Server thread/INFO] [Mohist]: [Plan]   ██▌██▌██▌██▌  v5.4 build 1516
[04:07:24] [Server thread/INFO] [Mohist]: [Plan] 
[04:07:24] [Server thread/WARN] [Mohist]: [Plan] ! ------- Deprecation warning ------- !
[04:07:24] [Server thread/WARN] [Mohist]: [Plan] Plan version 5.5 will require Java 11 or newer,
[04:07:24] [Server thread/WARN] [Mohist]: [Plan] consider updating your JVM as soon as possible.
[04:07:24] [Server thread/WARN] [Mohist]: [Plan] ! ----------------------------------- !
[04:07:24] [Server thread/INFO] [Mohist]: [Plan] Locale: 'English' by AuroraLS3
[04:07:24] [Server thread/INFO] [Mohist]: [Plan] Downloading MySQL Driver, this may take a while...
[04:07:24] [Server thread/ERROR] [Mohist]: Error occurred while enabling Plan v5.4 build 1516 (Is it up to date?)
java.lang.NoClassDefFoundError: com/mohistmc/bukkit/nms/proxy/ProxyClass
	at com.mysql.cj.jdbc.NonRegisteringDriver.<clinit>(NonRegisteringDriver.java:98) ~[mysql-mysql-connector-java-8.0.27.jar:?]
	at sun.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method) ~[?:1.8.0_312]
	at sun.reflect.NativeConstructorAccessorImpl.newInstance(NativeConstructorAccessorImpl.java:62) ~[?:1.8.0_312]
	at sun.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45) ~[?:1.8.0_312]
	at java.lang.reflect.Constructor.newInstance(Constructor.java:423) ~[?:1.8.0_312]
	at plan.com.zaxxer.hikari.HikariConfig.setDriverClassName(HikariConfig.java:495) ~[Plan-5.4-build-1516.jar:?]
	at com.djrapitops.plan.storage.database.MySQLDB.setupDataSource(MySQLDB.java:117) ~[Plan-5.4-build-1516.jar:?]
	at com.djrapitops.plan.storage.database.SQLDB.init(SQLDB.java:132) ~[Plan-5.4-build-1516.jar:?]
	at com.djrapitops.plan.storage.database.DBSystem.enable(DBSystem.java:92) ~[Plan-5.4-build-1516.jar:?]
	at com.djrapitops.plan.storage.database.BukkitDBSystem.enable(BukkitDBSystem.java:56) ~[Plan-5.4-build-1516.jar:?]
	at com.djrapitops.plan.PlanSystem.enableSystems(PlanSystem.java:201) ~[Plan-5.4-build-1516.jar:?]
	at com.djrapitops.plan.PlanSystem.enable(PlanSystem.java:172) ~[Plan-5.4-build-1516.jar:?]
	at com.djrapitops.plan.Plan.onEnable(Plan.java:77) ~[?:?]
	at org.bukkit.plugin.java.JavaPlugin.setEnabled(JavaPlugin.java:177) ~[JavaPlugin.class:?]
	at org.bukkit.plugin.java.JavaPluginLoader.enablePlugin(JavaPluginLoader.java:318) [JavaPluginLoader.class:?]
	at org.bukkit.plugin.SimplePluginManager.enablePlugin(SimplePluginManager.java:399) [SimplePluginManager.class:?]
	at org.bukkit.craftbukkit.v1_12_R1.CraftServer.enablePlugin(CraftServer.java:477) [CraftServer.class:?]
	at org.bukkit.craftbukkit.v1_12_R1.CraftServer.enablePlugins(CraftServer.java:418) [CraftServer.class:?]
	at net.minecraft.server.MinecraftServer.func_71247_a(MinecraftServer.java:383) [MinecraftServer.class:?]
	at net.minecraft.server.dedicated.DedicatedServer.func_71197_b(DedicatedServer.java:315) [nz.class:?]
	at net.minecraft.server.MinecraftServer.run(MinecraftServer.java:590) [MinecraftServer.class:?]
	at java.lang.Thread.run(Thread.java:748) [?:1.8.0_312]
```

### Plugin versions

Plan v5.4 build 1516
Mohist 1.12.2-292

**Comment by austin4195** [comment]
Issue resolved in Mohist 1.12.2 Build 299

**Comment by AuroraLS3** [comment]
Thanks!
